testscommon/factory/validatorStatisticsProcessorFactoryStub.go

```diff
@@ -0,0 +1,29 @@
+package factory
+
+import (
+	"github.com/multiversx/mx-chain-go/process"
+	"github.com/multiversx/mx-chain-go/process/peer"
+)
+
+// ValidatorStatisticsProcessorFactoryStub -
+type ValidatorStatisticsProcessorFactoryStub struct {
+	CreateValidatorStatisticsProcessorCalled func(args peer.ArgValidatorStatisticsProcessor) (process.ValidatorStatisticsProcessor, error)
+}
+
+// NewValidatorStatisticsProcessorFactoryStub -
+func NewValidatorStatisticsProcessorFactoryStub() *ValidatorStatisticsProcessorFactoryStub {
```

Comment:
Why the need for these constructors on all the stubs? These are stubs and are anyway exported structs

Reply:
no need, they were just "eye-candy", looked a little bit better... removed